WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift

```diff
@@ -572,6 +573,14 @@ extension SiteStatsInsightsTableViewController: SiteStatsInsightsDelegate {
         viewNeedsUpdating = false
     }
 
+    func scrollToNewCard() {
+        let lastSection = max(tableView.numberOfSections - 1, 0)
+
+        // newly added card will be penultimate row
```

Comment:
We could add `, above the 'Add Stats Card' row` just to clarify why it's the penultimate one